examples/psql/schema.sql

```diff
@@ -0,0 +1,69 @@
+CREATE TABLE IF NOT EXISTS benchmarks (
+    name TEXT PRIMARY KEY
+);
+
+CREATE TABLE IF NOT EXISTS branches (
+    name TEXT PRIMARY KEY
+);
+
+CREATE TABLE IF NOT EXISTS stores (
+    name TEXT PRIMARY KEY
+);
+
+CREATE TABLE IF NOT EXISTS instance_types (
+    name TEXT PRIMARY KEY
+);
+
+CREATE TABLE IF NOT EXISTS scenarios (
+    name TEXT PRIMARY KEY
+);
+
```

Comment:
Are the above single column foreign key tables really necessary / worth it?
... said the MongoDB guy. :-)

Who and when maintains the content of above tables? What happens if CI ran a test called guggenheimir, and such a benchmark doesn't exist in above table? Is the input data / test result discarded and why is that  a good thing?

Reply:
I agree that these foreign key tables distract from the main goal of this example - to show how to wire Hunter with PostgreSQL; removed in [d905714](https://github.com/Gerrrr/hunter/pull/13/commits/d90571424a17da8234fce1dc1dfbb356dacae79e).

For context, I initially added these FK tables for Kafka Streams benchmarks to ensure correct parsing/loading of benchmark configuration from previously used storage - raw HTML files. With tests upserting their configuration to these FK tables, it was easy to audit and catch parsing errors like `guggenheimir` and `guggenheimir ` being treated as different tests.